### ISSUE TITLE
Implement the Evaluation techniques

### ISSUE THREAD

**Issue by marcelcaraciolo** [muricoca/crab#53]
Implement the recommender evaluation techniques:  MAE, RMSE, F1-Score, Precision and Recall.

Evaluating collaborative filtering recommender systems
References:
http://dl.acm.org/citation.cfm?id=963772

Evaluation of recommender systems: new approach

http://ec.iem.cyut.edu.tw/drupal/sites/default/files/Evaluation%20of%20recommender%20systems%20A%20new%20approach.pdf

**Comment by marcelcaraciolo** [comment]
https://github.com/marcelcaraciolo/crab/blob/master/crab/evaluation/statistics.py
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/metrics/metrics.py

**Comment by marcelcaraciolo** [comment]
Added also the normalized mean absolute error  NMAE.

**Comment by marcelcaraciolo** [comment]
Implemeted the metrics MAE, RMSE, F1-Score, Precision and Recall
Check the Pull #57

**Comment by marcelcaraciolo** [comment]
Merged.
